modules/users/server/controllers/users.authentication.server.controller.js

```diff
@@ -84,6 +84,20 @@ exports.signup = function (req, res) {
 
     },
 
+    // Report how did you hear about us to stats
+    function () {
```

Comment:
I'm no async wizard ✨  either but it Looks like this function should receive a callback var `done`, which then should be called in `statService.stat`'s callback, I'm testing this now in #970.